ci/README.md

```diff
@@ -76,7 +76,7 @@ verifying that the container stops shortly afterwards by running docker ps.
 ## Add a platform
 
 To add a platform, you should add the appropriate dockerfile in
-docker/Dockerfile.build.<platform> and add a shell function named
+docker/Dockerfile.<platform> and add a shell function named
```

Comment:
Renaming the files should be a separate change if it's needed

Reply:
https://github.com/apache/incubator-mxnet/pull/12423